events/elections/2019/README.md

```diff
@@ -0,0 +1,151 @@
+# 2019 VOTERS GUIDE - KUBERNETES STEERING COMMITTEE ELECTION
+
+## Purpose
+
+The role of this election is to fill out the four (4) seats due for
+reelection this year on the [Kubernetes Steering Committee]. Each elected
+member will serve a two (2) year term.
+
+## Background
+
+This election will shape the future of Kubernetes as a community and project.
+While SIGs and WGs help shape the technical direction of the project, the
+[Steering Committee Charter] covers the health of the project and community
+as a whole. Some examples of responsibilities to consider as you are deciding
+whether to run or who to vote for:
+
+- Define, evolve, and defend the vision, values, mission, and scope of the
```

Comment:
Please update this to reflect the updated bullet(s) in the charter:
https://github.com/kubernetes/steering/blob/master/charter.md#direct-responsibilities-of-the-steering-committee